portfolio/src/main/webapp/index.html

```diff
@@ -25,6 +25,27 @@ <h1 id="title">My Portfolio</h1>
       <div id="greeting-container"></div>
     
     <img src="/images/city-birdeye.jpg" id="img-container"/>
+
+    <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+        <input type="submit" />
+    </form>
+
+    <form action="/contact" method="POST">
+        <p>Name:</p>
+        <textarea name="input-name">Your Name Here</textarea>
+        <p>Email:</p>
+        <textarea name="input-email">Your Email Here</textarea>
+        <p>Phone Number</p>
+        <textarea name="input-number">Your Phone Number Here</textarea>
+        <p>Got Beef? Leave it here.</p>
+        <textarea name="input-jokes">Got jokes or beef, leave it here</textarea>
```

Comment:
Tip on `name`: these are all going to be encoded in a POST request. It's a bit redundant to prefix all of them with `input-`.